scripts/deploy.sh

```diff
@@ -22,10 +22,12 @@ export REGISTRY=quay.io/kubernetes-service-catalog/
 
 docker login -e="${QUAY_EMAIL}" -u "${QUAY_USERNAME}" -p "${QUAY_PASSWORD}" quay.io
 
-if [[ -n "${TRAVIS_TAG}" ]]; then
+if [[ "${TRAVIS_TAG}" =~ ^v[0-9]\.[0-9]\.[0-9][a-z]*$ ]]; then
     echo "Pushing images with tag ${TRAVIS_TAG}."
     VERSION="${TRAVIS_TAG}" make push
-elif [[ "${TRAVIS_PULL_REQUEST}" == "false" && "${TRAVIS_BRANCH}" == "master" ]]; then
+elif [[ "${TRAVIS_BRANCH}" == "master" ]]; then
```

Comment:
Good point. Ok, I'm game for trying this. Wanna try [dpl](https://github.com/travis-ci/dpl) on this to see if it works as expected?